pages/linux/broot.md

```diff
@@ -0,0 +1,28 @@
+# broot
+
+> A better way to navigate directories.
+> More information: <https://github.com/Canop/broot>.
+
+- Install or reinstall the br shell function:
+
+`broot --install`
+
+- Navigate the directory:
+
+`br`
+
+- Display the size of files and directories:
```

Comment:
Is this of all directories on the file system or just ones in the current working directory.

Reply:
> Is this of all directories on the file system or just ones in the current working directory.

@CleanMachine1 It opens up at the current working directory but, you can navigate its child & even parent directory as it's a TUI program.